tests/unit/tf/core/test_combinators.py

```diff
@@ -251,7 +251,7 @@ def test_with_model(self, run_eagerly, music_streaming_data):
             layer,
             tf.keras.layers.Dense(1),
             mm.BinaryClassificationTask("click"),
-            schema=music_streaming_data.schema,
+            schema=music_streaming_data.schema.select_by_name("item_recency"),
```

Comment:
This test started failing after the change to `Model.from_config` because we were passing in an inconsistent schema to the model compared with what it expected.